layouts/partials/footer.html

```diff
@@ -14,3 +15,4 @@
         </span>
     </p>
 </footer>
+{{ end }}
```

Comment:
This doesn't look quite right to me: If the page is not a page, then the footer tag will not be closed. So the `</footer>` should at least be after the `{{ end }}`. But I suspect even more could be moved out of the if-block, so that we still have some contents in the footer of "non-page pages".

Reply:
Good point, I wasn't paying attention. I changed it so that only the relevant part is inside the if-block.